README.md

```diff
@@ -167,22 +146,108 @@ You can also make a package request by creating a [new issue](https://github.com
 
 You can also [create a package yourself](https://github.com/luxtorpeda-dev/packages/blob/master/docs/Creating_a_Package.md)
 
-## Development
+## Other installation methods
+
+### Pre-requisites
+
+Compatibility tool will work on any modern 64-bit Linux distribution.
+All packaged games are compiled for Steam Runtime Sniper environment and any additional dependencies are provided in the download.
+
+**Using Luxtorpeda with [Steam native runtime](https://wiki.archlinux.org/index.php/Steam/Troubleshooting#Steam_native_runtime) may or may not work, but is not supported.**
```

Comment:
These details  should be noted in general further above, as it's not directly related to how luxtorpeda is put in the right spot.